### PR TITLE
Feature/preferences endpoint

### PR DESCRIPTION
**Pull request by chubchenko**

### CODE REVIEW

**Comment by GeorgeZhukov** [comment]
ok

File: spec/support/matchers/serialize_object.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+RSpec::Matchers.define :serialize_object do |expected|
```
Comment (vshmyhlo) [inline spec/support/matchers/serialize_object.rb:1]: @Chubchenko, this is :+1:, can you also use it in specs for #update controller method?

Comment (chubchenko) [inline spec/support/matchers/serialize_object.rb:1]: @v-shmyhlo fixed

**Comment by RomanMukhin** [comment]
+

**Pull request merged**